src/test/java/sorinfoca/Tests/Tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,100 @@
+package sorinfoca.Tests.Tema6;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import sorinfoca.driver.BrowserManager;
+import java.io.File;
+import java.io.IOException;
+import java.util.Set;
+
+public class DemoQaWindow {
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQaWindowPage();
+        try {
+            clickNewTabButton();
+            interactWithElementsOnParentTab();
+        } catch (NoSuchElementException e) {
+            File screenshot = driver.getScreenshotAs(OutputType.FILE);
```

Comment:
All the lines written in this catch clause are already written in FileUtils.takeScreenshot() method. Use that method instead of all these lines as used in HerokuAlerts class. Thanks!